### PR TITLE
Fixes based on dashboard feedback

### PR DESCRIPTION
**Pull request by ewanmay**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -2,18 +2,16 @@ import React, { useContext, useEffect, useState } from "react";
 import { Link, Route, Switch, useLocation, Redirect } from "react-router-dom";
 import './App.css';
 import Icon from './assets/images/two-tone-light.svg';
-import Data from "./components/pages/Data";
+import Dashboard from "./components/pages/Data";
 import About from './components/static/About';
 import Auth from './components/static/Auth/Auth';
-import Home from './components/static/Home';
+import Data from './components/static/Home';
```
Comment (atatmaja) [inline src/App.tsx:8]: So as I understand, what was formerly called the Home tab will now be called the Data tab right? Is this something Rahul is expecting or nah?

File: src/components/charts/Charts.tsx [hunk 2]
```diff
@@ -43,17 +52,21 @@ export default function Charts() {
...
-    const { x, y, width, height, value } = props;
+    const { x, y, width, height, value, index } = props;
+    let recordError = records[index].error;
+    const ratio = width / value;
+    const error = Array.isArray(recordError) ? recordError[1] : recordError;
+    const errorBarWidth = ratio * error;
     return (
       <g>
-        <text x={x + width + 10} y={y + height / 2} fill="#000" textAnchor="right" dominantBaseline="right">
+        <text x={x + width + 10 + (errorBarWidth as number)} y={y + height / 2 + 5} fill="#000" textAnchor="right" dominantBaseline="right">
```
Comment (atatmaja) [inline src/components/charts/Charts.tsx:62]: better to type errorBarWidth in its definition or nah?

File: src/App.tsx (hunk 1 above)
Comment (rahularoradfs) [reply src/App.tsx:8]: yep! discussed

File: src/components/map/Legend.tsx [hunk 3]
```diff
@@ -15,20 +16,32 @@ const Legend = (props: legendProps) => {
     const control = L.control as any
     const legend = control({ position: "bottomright" });
     // apparently this is not caught in the typings so I have to any it
-    L.DomUtil.remove(legend)
     legend.onAdd = () => {
       const div = L.DomUtil.create("div", "info flex legend center-item");
-      const title = '<h4 className="legend-title col-12 p-0 flex middle">Seroprevalence (logit scale) </h4>'
+      if (isNaN(buckets[1])) {
+        return div
```
Comment (atatmaja) [inline src/components/map/Legend.tsx:22]: should we return some message instead of nothing here?

**Pull request merged**